src/components/commands/layout.tsx

```
@@ -5,461 +5,679 @@ import { ChevronDown, Search, Menu, X } from 'lucide-react';
+                                </Badge>
+                              ))}
+                            </div>
+                          </div>
+                        )}
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              )}
+              
+              {botState?.command_permissions && botState.command_permissions[command.qualified_name || command.name] && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)